lib/home/view/tasks_page.dart

```diff
@@ -12,8 +14,14 @@ class TasksPage extends StatelessWidget {
       appBar: AppBar(
         title: const Text('To Do List'),
       ),
-      body: const TaskListItem(),
-      
-    );
+      body: Center(
+        child: Container(
+          padding: const EdgeInsets.all(20),
+          height: 200,
+          width: 400,
+          child: TaskCard(task: TasksDevData.tasks[0]),
```

Comment:
you can add comment here just to remember to change it later. e.g:
`TODO(kamil): for dev purposes only`